src/operation.ts

```diff
@@ -566,7 +566,3 @@ export class Callback extends Operation {
 }
 
 export class Webhook extends Operation {}
-
-module.exports = Operation;
-module.exports.Callback = Callback;
-module.exports.Webhook = Webhook;
```

Comment:
Not sure what happened here but some changes to the [import/no-import-module-exports](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-import-module-exports.md) rule caught it.